internal/main_vm.go

```diff
@@ -31,6 +34,27 @@ func Main() {
 	}
 }
 
+// Find the path to package main by looking at the root Caller.
+func findMainPath() string {
+	pc := make([]uintptr, 20)
+	_ = runtime.Callers(0, pc)
```

Comment:
You need to keep the return value so you know how many entries have been filled in. Since you know it won't be this function, you can skip elements.

```suggestion
	n := runtime.Callers(2, pc)
```